src/ts/components/MatchOverlay.tsx

```diff
@@ -69,6 +75,21 @@ const matchOverlay = <TPluginState extends IPluginState>({
     debounceShowMatch.current(true)
   }, [currentMatchId]);
 
+  const getOffsets = React.useMemo(() => ({ placement }: {placement: Placement}): [number | null | undefined, number | null | undefined] => {
+    const isTop = placement.indexOf("top") >= 0;
+    const isBottom = placement.indexOf("bottom") >= 0
+    if(referenceElement && currentRectIndex !== undefined && (isTop || isBottom)){
+      const rects = referenceElement?.getClientRects();
```

Comment:
Any comments here to clarify what's going on what be really welcome! I understand the y-axis process, but the x-axis ... I guess if we're hovering over the same rect as the last rect, the offset is 0, else we offset by the hoverRect – ah, I see, because the lastRect is _always_ where the tooltip is positioned for a `bottomLeft` tooltip?

Reply:
I've added some comments to this. It will only work at the moment due to the currently chosen placement. If we wanted to support other placements we would need to make some changes to how the X offset is calculated.

Reply:
Ah brill, yes nice to make the dependency explicit, if only in text.